### ISSUE TITLE
wav2vec still working?

### ISSUE THREAD

**Issue by tngt** [Fictionarry/TalkingGaussian#6]
very great work, wav2vec still working?

**Comment by Fictionarry** [comment]
Yes,  the processing code for wav2vec is preserved, following a similar preprocessing and usage as in [ER-NeRF](https://github.com/Fictionarry/ER-NeRF)
